integration-tests/src/test/java/tachyon/client/FileOutStreamIntegrationTest.java

```diff
@@ -74,39 +72,38 @@ public final void after() throws Exception {
 
   @Before
   public final void before() throws Exception {
-    TachyonConf tachyonConf = WorkerContext.getConf();
-    tachyonConf.set(Constants.USER_FILE_BUFFER_BYTES, String.valueOf(BUFFER_BYTES));
-    // Only the Netty data server supports remote writes.
-    tachyonConf.set(Constants.WORKER_DATA_SERVER, IntegrationTestConstants.NETTY_DATA_SERVER);
     sLocalTachyonCluster.start();
     mTfs = sLocalTachyonCluster.getClient();
-    mMasterTachyonConf = sLocalTachyonCluster.getMasterTachyonConf();
+  }
+
+  @BeforeClass
+  public static void beforeClass() throws IOException {
+    sLocalTachyonCluster =
+        new LocalTachyonCluster(WORKER_CAPACITY_BYTES, QUOTA_UNIT_BYTES, BLOCK_SIZE_BYTES);
+    sTestConf = sLocalTachyonCluster.getTestConf();
+    sTestConf.set(Constants.USER_FILE_BUFFER_BYTES, String.valueOf(BUFFER_BYTES));
+    // Only the Netty data server supports remote writes.
+    sTestConf.set(Constants.WORKER_DATA_SERVER, IntegrationTestConstants.NETTY_DATA_SERVER);
     sWriteBoth =
-        new OutStreamOptions.Builder(mMasterTachyonConf)
+        new OutStreamOptions.Builder(sTestConf)
```

Comment:
Hmm I think passing in conf to the builder is a mistake if we are assuming the conf is the same as `ClientContext.getConf`, but this is definitely not in the scope of this PR.